src/SlippiGame.ts

```diff
@@ -1,4 +1,5 @@
 import type { StatOptions, StatsType } from "./stats";
+import { HomerunComputer, TargetBreakComputer } from "./stats";
```

Comment:
```suggestion
import { HomeRunComputer, TargetBreakComputer } from "./stats";
```
Can we capitalise the `Run` in `homeRun` and `HomeRun`?